### ISSUE TITLE
Unparseable row externs being generated

### ISSUE THREAD

**Issue by garyb** [purescript/purescript#1084]
Currently, this: https://github.com/slamdata/slamdata/blob/b48281172de1fa232b6deb3267785525ff01a5aa/src/Entries/Notebook.purs

Generates:
``` purescript
-- Generated by psc-make version 0.6.9.5
module Entries.Notebook where
import Prelude ()
import Utils ()
import Halogen ()
import App.Notebook ()
import Driver.Notebook ()
import Prim ()
import Prelude ()
import App.Notebook ()
import Data.Tuple ()
import Halogen ()
import Utils ()
import Driver.Notebook ()
foreign import main :: forall t29. Control.Monad.Eff.Eff (trace :: Debug.Trace.Trace, ref :: Control.Monad.Eff.Ref.Ref, dom :: DOM.DOM | EffectTypes.NotebookComponentEff t29) Prelude.Unit
```

Which results in this:
```
"dist/node_modules/Entries.Notebook/externs.purs" (line 15, column 58):
unexpected LParen
expecting no indentation or end of input
```

**Comment by paf31** [comment]
I was thinking today - maybe we could just use `Show` and `Read` for externs. Is there really a need for them to be valid PS files?

**Comment by paf31** [comment]
Is it possible to add a type signature to work around this for now?

**Comment by garyb** [comment]
Yep, that works fine.

**Comment by paf31** [comment]
Should be fixed by #1480, but I haven't checked.
